content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,332 @@
+
+.. image:: create_quotations/quotation-form.png
+   :align: center
+   :alt: A typical quotation form in the Odoo Sales application.
+
+Begin by entering the customer's name in the :guilabel:`Customer` field at the top of the form. This
+is a **required** field.
+
+If the customer's information is already in the database, the :guilabel:`Invoice Address` and
+:guilabel:`Delivery Address` fields auto-populate with the saved information for those respective
+fields, based on the data from that customer's contact record (found in the *Contacts* application).
+
+If the customer was referred by another customer or contact, enter their name in the
+:guilabel:`Referrer` field.
+
+If a :guilabel:`Referrer` is selected, a new field, :guilabel:`Commission Plan` appears, in which
+a commission can be selected from the drop-down menu. This commission is rewarded to the contact
+selected in the :guilabel:`Referrer` field.
+
+Next, if they have not already been auto-populated with the customer's information, enter the
+appropriate addresses in the :guilabel:`Invoice Address` and :guilabel:`Delivery Address` fields.
+Both of these fields are **required**.
+
+Then, if desired, choose a :guilabel:`Quotation Template` from the drop-down field to apply to this
+quotation.
+
+The default date that appears in the :guilabel:`Expiration` field is based on the number configured
+in the :guilabel:`Default Quotation Validity` setting (in :menuselection:`Sales app -->
+Configuration --> Settings`).
+
+.. tip::
+   When using a quotation template, the date in the :guilabel:`Expiration` field is based off the
+   :guilabel:`Quotation Validity` figure on the template form.
+
+If the quotation is for a recurring product or subscription, select the desired :guilabel:`Recurring
+Plan` from that specific drop-down menu.
+
+If desired, select a specific :guilabel:`Pricelist` to be applied to this quotation.
+
+Lastly, select any specific :guilabel:`Payment Terms` to be used for this quotation.
+
+Order Lines tab
+---------------
+
+The first tab on the quotation form is the :guilabel:`Order Lines` tab.
+
+In this tab, select products, and quantities of those products, to add them to the quotation.
+
+There are two ways to add products to the quotation from this tab.
+
+Clicking :guilabel:`Add a product`, selecting the desired item from the :guilabel:`Product`
+drop-down field, and proceeding to adjust the quantity of that selected product, if necessary, is
+one way to add products to a quotation.
+
+Or, click :guilabel:`Catalog` to reveal a separate page, showcasing every item (and every potential
+product variant) in a neatly organized display, with items organizable by :guilabel:`Product
+Category` and :guilabel:`Attributes`.
+
+.. image:: create_quotations/product-catalog.png
+   :align: center
+   :alt: A product catalog accessible via a quotation in the Odoo Sales application.
+
+From here, simply locate the desired items, click the :icon:`fa-shopping-cart` :guilabel:`Add`
```

Comment:
You don't have to click the cart- clicking anywhere on the card adds more items. This confused me since I clicked on a card to reveal the product details, and it added it to the cart!  Kind of odd to even have a cart button at that point =D
Just wanted to mention it!

Reply:
Oooh that *is* interesting. Great catch! I'm assuming this **might** be a bug that is fixed later...? Not sure. Either way, I think if I keep this as-is, I won't have to return to it down the line to specify that the 'Add' button needs to be clicked. At this point, though, like you said, clicking anywhere on the card adds it, so users may see that as a "happy accident" haha